containers/images/redis/Dockerfile.exporter

```diff
@@ -0,0 +1,32 @@
+FROM docker.io/sourcemation/debian-12-slim:latest as builder
```

Comment:
This should be separate image, you can skip test as it is compiled external binay, look at conf.sh of rabbitmq-cluster-operator https://github.com/SourceMation/images/blob/main/containers/images/rabbitmq-cluster-operator/conf.sh